deepfence_worker/cronjobs/cloud_compliance.go

```diff
@@ -237,9 +238,9 @@ func CachePostureProviders(ctx context.Context, task *asynq.Task) error {
 
 			scan_count_query = `
 			MATCH (n:` + string(neo4jNodeType) + `)
-			WHERE n.pseudo=false and n.active=true and n.agent_running=true
```

Comment:
Is the condition `n.agent_running=true` not required for linux posture count too?

Reply:
hosts with agent_running=false are not allowed to scan so they will not appear here